### PR TITLE
fix benchmarks for greater than 256

### PR DESCRIPTION
**Pull request by justinzhou93**

### CODE REVIEW

File: runtime/neumann/src/lib.rs [hunk 1]
```diff
@@ -727,7 +727,7 @@ impl pallet_democracy::Config for Runtime {
 }
 
 parameter_types! {
-	pub const MaxTasksPerSlot: u32 = 256;
+	pub const MaxTasksPerSlot: u32 = 576;
```
Comment (justinzhou93) [inline runtime/neumann/src/lib.rs:730]: don't have to bump spec_version in this PR, right? we'll just do in the next release

File: pallets/automation-time/src/weights.rs [hunk 2]
```diff
@@ -106,7 +106,7 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
 	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
 	// Storage: AutomationTime ScheduledTasks (r:24 w:24)
 	fn cancel_scheduled_task_full() -> Weight {
-		(542_000_000 as Weight)
+		(1_192_000_000 as Weight)
```
Comment (justinzhou93) [inline pallets/automation-time/src/weights.rs:109]: lordie this got heavy

File: pallets/automation-time/src/weights.rs [hunk 3]
```diff
@@ -124,7 +124,7 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
 	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
 	// Storage: AutomationTime ScheduledTasks (r:24 w:24)
 	fn force_cancel_scheduled_task_full() -> Weight {
-		(524_000_000 as Weight)
+		(1_203_000_000 as Weight)
```
Comment (justinzhou93) [inline pallets/automation-time/src/weights.rs:127]: this too. damn, cancels got heavy

**Comment by justinzhou93** [comment]
key here is turning an array of a single u8 number into an array of 2 u8 numbers. since (u8, u8) should offer the same number of probabilities as u16 (2^8 * 2^8 = 2^16)

**Review by lreesby — APPROVED** [review]
lgtm

**Pull request merged**